src/contexts/IpfsStorageContext.js

```diff
@@ -142,12 +144,11 @@ const getTemporalNode = async (username, password) => {
       })
       return response.json()
     },
-    dagPut: async () => {
-      let key = Math.random().toString()
-      let val = Math.random().toString()
-      let blob = new Blob([{ [key]: val }], {
+    dagPut: async json => {
+      let blob = new Blob([JSON.stringify([json])], {
```

Comment:
@radio-alice this is ok for this PR, since we're making changes in the `feat/storage-tab` branch that are related to this (and this PR is getting merged into `feat/storage-tab`), but next time make sure to only include code changes that relate to your commits

Reply:
oops good point- when would be the right time to drop the test code then?